manifest_staging/charts/csi-secrets-store-provider-azure/templates/arc-monitoring.yaml

```diff
@@ -0,0 +1,213 @@
+{{- if .Values.enableArcExtension }}
+{{- if .Values.arc.enableMonitoring }}
+{{- if .Values.linux.enabled}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: arc-{{ template "sscdpa.fullname" . }}-monitoring
```

Comment:
What if this name is > 63 chars? That would result in an error at the deployment. Should we use static names here instead?

Reply:
updated.